mm2src/coins/eth/v2_activation.rs

```diff
@@ -662,6 +663,9 @@ pub(crate) async fn build_address_and_priv_key_policy(
                 .map_to_mm(|e| EthActivationV2Error::InternalError(e.to_string()))?;
             let bip39_secp_priv_key = global_hd_ctx.root_priv_key().clone();
 
+            #[cfg(not(target_arch = "wasm32"))]
+            run_db_migration_for_new_eth_pubkey(ctx, dhash160(activated_key.public().as_bytes()).to_string()).await?;
```

Comment:
This should also use the same `db_id` as `eth_account_db_id` stated in my first comment.

Reply:
done